### ISSUE TITLE
How's the same conn possible?

### ISSUE THREAD

**Issue by moscovium0** [haxball/haxball-issues#2097]
I have a log system that logs player auth and their conn. I've just realized that sometimes players get the same conn. How's that possible?

**Comment by damian18zb** [comment]
Maybe they are living together, so their ip is same.

**Comment by moscovium0** [comment]
No, it was me. I was checking logs and I saw another player has different auth but same conn as me. I'm terrified for real. Its only me in the house living, and I only use the network. How's that possible?

**Comment by moscovium0** [comment]
We use same internet provider tho ^^
